scripts/base/init-bare.zeek

```diff
@@ -6138,59 +6179,7 @@ export {
 
 module GLOBAL;
 
-## Seed for hashes computed internally for probabilistic data structures. Using
-## the same value here will make the hashes compatible between independent Zeek
-## instances. If left unset, Zeek will use a temporary local seed.
-const global_hash_seed: string = "" &redef;
-
-## Number of bits in UIDs that are generated to identify connections and
-## files.  The larger the value, the more confidence in UID uniqueness.
-## The maximum is currently 128 bits.
-const bits_per_uid: count = 96 &redef;
-
-## This salt value is used for several message digests in Zeek. We
-## use a salt to help mitigate the possibility of an attacker
-## manipulating source data to, e.g., mount complexity attacks or
-## cause ID collisions.
-## This salt is, for example, used by :zeek:see:`get_file_handle`
-## to generate installation-unique file IDs (the *id* field of :zeek:see:`fa_file`).
-const digest_salt = "Please change this value." &redef;
-
-## Maximum string length allowed for calls to the :zeek:see:`find_all` and
-## :zeek:see:`find_all_ordered` BIFs.
-const max_find_all_string_length: int = 10000 &redef;
-
-## How many rounds to go without checking IO sources with file descriptors
-## for readiness by default. This is used when reading from traces.
-##
-## Very roughly, when reading from a pcap, setting this to 100 results in
-## 100 packets being processed without checking FD based IO sources.
-##
-## .. note:: This should not be changed outside of development or when
-##    debugging problems with the main-loop, or developing features with
-##    tight main-loop interaction.
-##
-## .. zeek:see:: io_poll_interval_live
-const io_poll_interval_default = 100 &redef;
-
-## How often to check IO sources with file descriptors for readiness when
-## monitoring with a live packet source.
-##
-## The poll interval gets defaulted to 100 which is good for cases like reading
-## from pcap files and when there isn't a packet source, but is a little too
-## infrequent for live sources (especially fast live sources). Set it down a
-## little bit for those sources.
-##
-## .. note:: This should not be changed outside of development or when
-##    debugging problems with the main-loop, or developing features with
-##    tight main-loop interaction.
-##
-## .. zeek:see:: io_poll_interval_default
-const io_poll_interval_live = 10 &redef;
-
-## Whether Zeek is being run under test. This can be used to alter functionality
-## while testing, but should be used sparingly.
-const running_under_test: bool = F &redef;
+@load base/bif/event.bif
 
 global done_with_network = F;
```

Comment:
Also wondering if the state variable could be deprecated, but feel free to disregard.

Reply:
From a quick grep, nothing is using it.